packages/trader/src/Modules/CFD/Components/cfd-account-card.tsx

```diff
@@ -252,8 +252,10 @@ const CFDAccountCard = ({
     };
 
     const handleClickSwitchAccount: () => void = () => {
-        toggleShouldShowRealAccountsList(true);
-        toggleAccountsDialog(true);
+        if (toggleShouldShowRealAccountsList && toggleAccountsDialog) {
+            toggleShouldShowRealAccountsList(true);
+            toggleAccountsDialog(true);
+        }
```

Comment:
```suggestion
        toggleShouldShowRealAccountsList?.(true);
        toggleAccountsDialog?.(true);
```
you can do optional call instead of having a if check